projects/elife.yaml

```diff
@@ -2169,3 +2169,27 @@ microsimulation-demo:
         ports:
             1280: 80
             1281: 8081
+
+bioprotocol:
+    repo: https://github.com/elifesciences/bioprotocol-service
+    aws:
+        type: t2.micro
+        ports:
+            - 22
+            - 80
+            - 443
+        sns:
+            # endpoint for BioProtocol to publish updates to article protocol data
+            - bioprotocol-updates--{instance}
```

Comment:
https://github.com/elifesciences/issues/issues/4930#issuecomment-498144947 states they would be writing to SQS; it doesn't make much of a difference but has this changed?

Reply:
I thought that was a mistake. Writing to a queue doesn't seem correct to me. Writing to a topic and having queues subscribed to those topics seems more-correct. Or I could just write an API endpoint they can PUT/POST to directly. I'll follow this point up with them

Reply:
I've contacted Dong. He seems like a cheerful guy.

Reply:
Queues (SQS) are meant for 1-to-1 communication, whereas topics/fanouts (SNS) are meant for 1-to-N communication.

Reply:
I know, but that seems so very limited to almost be a side effect of the larger design AWS has with SNS+SQS. There is an awful lot I can do while I wait for Dong to confirm so no urgency.

Reply:
With different names, it is a general pattern of messaging systems:
https://www.enterpriseintegrationpatterns.com/patterns/messaging/PointToPointChannel.html
https://www.enterpriseintegrationpatterns.com/patterns/messaging/PublishSubscribeChannel.html

Reply:
yes, thank you Giorgio. (this is getting a little condescending, please stop.)

Reply:
they're happy to use POST. I've removed the SNS topic and SQS subscription